test/integration/features/integration.feature

```diff
@@ -202,7 +202,10 @@ Feature: Powermax OS CSI interface
     And max retries 1
     And a basic block volume request "integration4" "48"
     When I call CreateVolume
-    Then the error message should contain "OutOfRange desc = bad capacity"
```

Comment:
Are we no longer expected to see this error due to minimum capacity?

Reply:
I have made change to this test as according to our controller code : https://github.com/dell/csi-powermax/blob/main/service/controller.go#L1528
If volume size is less than minimum then it will create the volume with minimum size.
But this test was checking that volume should not be created .So I have changed this test according to our requirement .